test/unit/helpers/clearStore.ts

```diff
@@ -0,0 +1,5 @@
+import store from '../../../src/store'
```

Comment:
this uses the store from '../../../src/store' but helpers/createLiveStore.ts does not, how does that work?

Reply:
`createLiveStore` creates a new instance and can be used to inject a mocked version of the store.

The store-reference is global and should be "cleaned" (e.g. by using clearStore-helper) after each test that adds triples to it.

I prefer the former as it cannot lead to "bleeding tests" (i.e. tests that forgot to clean up after themselves), but we cannot use it all the time, as some functionality depends on the global store.

Thumb finger rule: If you can inject the store that should be worked on, prefer adding an instance created with `createLiveStore`.